src/app/joint-control/joint-control-core/joint-control-core.component.html

```diff
@@ -4,27 +4,35 @@
     [ngClass]="joint.reversed ? 'flex-row-reverse' : 'flex-row'">
     <div class="flex-grow-1 m-0 p-0">
         <div
-            class="row m-0 p-0 d-flex align-items-end"
+            class="row m-0 p-0 d-flex align-items-end pb-2"
             [ngClass]="joint.reversed ? 'flex-row-reverse' : 'flex-row'"
             [style]="'height: ' + ((joint.segmentOffset - joint.segmentHeight/2) * 100) + '%;'">
             <div class="col-3 sidebar-label m-0 p-0">Settings</div>
             <div class="col-3 sidebar-label m-0 p-0">Active</div>
             <div class="col-3 sidebar-label m-0 p-0">Current</div>
         </div>
+        <!--
+            In ngClass you can find the following statement: [motor.motorName == motorName && motor.motorName != 'all_fingers_left' && motor.sourceMotorName != 'all_fingers_right' || ( (motorName == 'all_fingers_left' || motorName == 'all_fingers_right') && motor.motorName != 'all_fingers_left' && motor.sourceMotorName != 'all_fingers_right') ? 'backgroundSelected' : ''  
+            This is used to higlite all rows when a motor is selected. This excludes the row of all_fingers_left and all_fingers_right
+        -->
         <div
             *ngFor="let motor of joint.motors"
             class="row m-0 p-0"
-            [ngClass]="joint.reversed ? 'flex-row-reverse' : 'flex-row'"
+            [ngClass]="[motor.motorName == motorName && motor.motorName != 'all_fingers_left' && motor.motorName != 'all_fingers_right' || ( (motorName == 'all_fingers_left' || motorName == 'all_fingers_right') && motor.motorName != 'all_fingers_left' && motor.motorName != 'all_fingers_right') ? 'backgroundSelected' : '',
```

Comment:
I think we should try to simplify this if-condition. We could do something like this:
`*ngFor="let motor of displayMotors"`
 `           class="row m-0 p-0"`
 `           [ngClass]="[motor.motorName == motorName || selectedMotor.displaySettings ? 'backgroundSelected' : '', ``joint.reversed ? 'flex-row-reverse' : 'flex-row']"`
In the ts-file we can compute `displayMotors` as `this.displayMotors = this.joint.motors.filter(motor => motor.displaySettings)` in the `data` subscription. The `selectedMotor` variable replaces the `motorName` variable and is an actual motor-configuration-object. If we do this, we can also remove the `<ng-container>` wrapper below. Also, the calculation of the if-condition should probably be moved to the ts-file and replaced with a variable in the html. If we do this approach we should probably also rename the `displaySettings` field to something more general like `isMultiMotor` or something similar.